libraries/classes/UserPassword.php

```diff
@@ -278,9 +279,11 @@ private function changePassDisplayPage($message, $sql_query)
             $sql_query,
             'success'
         );
-        echo '<a href="' , Url::getFromRoute('/')
-            , ' target="_parent">' , "\n"
-            , '<strong>' , __('Back') , '</strong></a>';
+        
+        $template = new Template();
+        echo $template->render('UserPassword', [
```

Comment:
We use snake case for template file names.
```suggestion
        echo $template->render('user_password', [
```

Reply:
fixed the Pr;excuse the silly mistakes...Is there any other change required?